### PR TITLE
Feature/home UI

### PR DESCRIPTION
**Pull request by Soha-Ahmed-Hamdy**
Design for Home Screen and Gradient Colors

### CODE REVIEW

File: iOS_Project/Module/Home/View/HomeViewController.swift [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        
+    
+        return cell
+    }
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        caterNames.count
+    }
+    
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        let cell = collectionView.dequeueReusableCell(withReuseIdentifier: "item", for: indexPath) as! CategoryCollectionViewCell
```
Comment (MinaKamal99) [inline iOS_Project/Module/Home/View/HomeViewController.swift]: please do not use force Unwrap here use OPTIONAL

File: iOS_Project/Module/Home/View/HomeViewController.swift [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+        
+    }
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        3
+        
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "FavouriteTableViewCell", for: indexPath) as! FavouriteTableViewCell
```
Comment (MinaKamal99) [inline iOS_Project/Module/Home/View/HomeViewController.swift]: please do not use force Unwrap here use OPTIONAL

File: iOS_Project/Module/Home/View/HomeViewController.swift (hunk 1 above)
Comment (Soha-Ahmed-Hamdy) [reply iOS_Project/Module/Home/View/HomeViewController.swift]: resolved

File: iOS_Project/Module/Home/View/HomeViewController.swift (hunk 2 above)
Comment (Soha-Ahmed-Hamdy) [reply iOS_Project/Module/Home/View/HomeViewController.swift]: resolved

**Review by MinaKamal99 — APPROVED** [review]

**Pull request merged**